addons/web/views/webclient_templates.xml

```diff
@@ -82,8 +82,9 @@
                 t-attf-href="http://www.odoo.com?utm_source=db&amp;utm_medium=#{_utm_medium}"
                 class="badge text-bg-light">
                 <img alt="Odoo"
-                        src="/web/static/img/odoo_logo_tiny.png"
-                        style="height: 1em; vertical-align: baseline;"/>
+                    src="/web/static/img/odoo_logo_tiny.png"
```

Comment:
Do we gain anything with this commit or is it just removing a "warning"?
I guess it can't be measured on pagespeed (due to non steady result) but maybe on chrome lighthouse?
Because it seems wrong at 2 level:
1. Without the `em` unit, it doesn't fit the page font size anymore
Before:
![image](https://user-images.githubusercontent.com/30048408/192162824-4fabe610-881d-4110-afe1-601a684a5e1a.png)
After:
![image](https://user-images.githubusercontent.com/30048408/192162836-9a6a1aa2-f468-47d8-a969-377aa00e8d10.png)
2. This one might not be that bad even if not ideal, I saw it directly because I was reviewing this image, not sure I would've seen it otherwise: the bottom of the letter are cut off. Obviously easier to see if you zoom, here is a 200% zoom for the illustration:
Before:
![image](https://user-images.githubusercontent.com/30048408/192162910-65e3bdaa-0196-4f9e-8244-622802328f7f.png)
After:
![image](https://user-images.githubusercontent.com/30048408/192162904-ea74f613-be49-4b4e-8de0-2b9132af80ac.png)

Reply:
> Do we gain anything with this commit or is it just removing a "warning"?
> I guess it can't be measured on pagespeed (due to non steady result) but maybe on chrome lighthouse?

Other than no layout shift as said in my commit message, I guess not. A nice enough reason to me.

> Without the em unit, it doesn't fit the page font size anymore

Of course. In fact, I guess there is no reason to not keep height: 1em and force width: auto. That way we'll have what I explain in the logo commit but without the compromise: it will already be good during loading the image. I should have thought of it :man_facepalming:  (but I did not not know about the subtilities of width/height attribute + css at the time, just investigated it for the logo commit).

> the bottom of the letter are cut off

Strange, I'll fix this commit tomorrow. Thanks for checking.

Reply:
> Other than no layout shift as said in my commit message, I guess not. A nice enough reason to me.

Sure I agree, but not if it means the logo is not following the font size IMO, but since you will fix it that's just win - win :)

Reply:
Again, thanks, it's of course better without compromise now, I was too quick :grimacing I made this diff and adapted the commit message: https://github.com/odoo/odoo/compare/da7b6b089821950d05e892ae4eb25cdbcf3325f8..618264b262e95776c0b59d90b1a0e9837fe04855

- Restore the height: 1em rule, combined with right width and height attribute + width auto, everything is perfect :+1: 
- Removed my attempt at optimizing the image -> in the end since the height: 1em is restored, the image could use that bigger size in the case the font-size is also bigger anyway